### ISSUE TITLE
Could you provide hyperparameters?

### ISSUE THREAD

**Issue by bybeye** [Kthyeon/FINE_official#5]
I found some hyperparameters are different from the description in the paper, "All hyper-parameters settings are the same with [25], even for the clean probability threshold.". For example, the threshold in the code is 0.6, but in DivideMix the threshold is 0.5. The number of warmup is also different from DivideMix. Could you provide hyperparameters for reproducing the results?
